app/src/main/res/values-az/strings.xml

```diff
@@ -56,6 +56,8 @@
         <b>Reddit:</b>
         https://www.reddit.com/r/SimpleMobileTools
     </string>
+    <string name="created_at">Created at</string>
```

Comment:
move the new strings into some proper place, not at the bottom

Reply:
Thanks for reviewing, I moved the new strings up to a different place. I also noticed some unnecessary leftover imports from my debugging, which I fixed as well. I would be glad if you could take another look.